SCClassLibrary/Common/Collections/SequenceableCollection.sc

```diff
@@ -129,6 +129,82 @@ SequenceableCollection : Collection {
 		^true
 	}
 
+	editDistance { | other, compareFunc |
```

Comment:
this was consistent with the preceding function in the file, but I see now that's the only one. Happy to change it.

if this is the preferred style, should I update the style-guide? It says spaces around binary ops, mentions to use pipes instead of arg, but doesn't mention spacing around pipes in arg lists?

Reply:
i'll update the style guide. i looked around, and actually we did reach consensus on this in https://github.com/supercollider/supercollider/issues/2913, just never documented it